types/shapediver/appbuildertypecheck.ts

```diff
@@ -136,6 +136,12 @@ const IAppBuilderWidgetPropsBarChartSchema = z.object({
 	style: z.enum([ "default", "stacked", "percent", "waterfall"]).optional(),
 }).extend(IAppBuilderWidgetPropsChartCommonSchema.shape);
 
+// Zod type definition for IAppBuilderWidgetPropsInteraction
+const IAppBuilderWidgetPropsInteractionSchema = z.object({
+	parameterName: z.string().optional(),
+	interactionSettings: z.any(),

```

Comment:
@MajorMeerkatThe3rd let's add the complete type definition here (we can limit this to the properties we use in the widget)